web/themes/custom/contribtracker/cypress.config.js

```diff
@@ -1,36 +1,9 @@
 import { defineConfig } from 'cypress';
-import { readFileSync } from 'fs';
-import path from 'path';
-// import cypressSplit from "cypress-split"; // Uncomment if you are using cypress-split
 
 export default defineConfig({
   projectId: 'contrib-tracker',
-  viewportWidth: 1512,
-  viewportHeight: 1200,
-  watchForFilesChanges: false,
-  chromeWebSecurity: false,
-  defaultCommandTimeout: 60000,
-  execTimeout: 10000,
-  pageLoadTimeout: 200000,
-  requestTimeout: 15000,
-  responseTimeout: 10000,
-  experimentalStudio: true,
-  trashAssetsBeforeRuns: false,
   video: process.env.CYPRESS_VIDEO !== 'false', // Defaults to true unless explicitly set to 'false'
-  includeShadowDom: true,
   e2e: {
-    async setupNodeEvents(on, config) {
-      const version = config.env.version || 'dev';
-      // Load env from json using dynamic import
-      const envConfigPath = path.resolve(`./cypress/config/${version}.json`);
-      const envConfig = JSON.parse(readFileSync(envConfigPath, 'utf-8'));
-      config.env = { ...config.env, ...envConfig };
-      // Change baseUrl
-      config.baseUrl = config.env.baseUrl;
-      console.log(config.baseUrl);
-      // cypressSplit(on, config); // Uncomment if you are using cypress-split
-      return config;
-    },
-    specPattern: 'cypress/e2e/**/*.{js,jsx,ts,tsx}',
+    baseUrl: process.env.CYPRESS_BASE_URL || 'https://contrib.axelerant.com', // Default to prod url if no base URL is set
```

Comment:
This environment variable needs to be set when run by DDEV as well. Otherwise, `ddev cypress-open` and `ddev cypress-run` will run tests against the live environment.

In fact, isn't it better to set the default to DDEV site and override it to production (or some other URL) if required?

Reply:
I have added the ddev url as env like below in yaml file:
`CYPRESS_BASE_URL: https://contribtracker.ddev.site/`

Shall I keep that field empty/comment out?

Reply:
@KosalaiV, the YAML file does not affect anything in DDEV. It will only affect the CI environment and even there, it will not be passed to DDEV.

Reply:
Ah, never mind. DDEV already sets this environment variable. This means we don't need the lines in `ci.yml` and `vr.yml`. In this case, the default of production URL is also fine, because DDEV will always set it otherwise.